proposals/clause_8_ogcapi_processes.adoc

```diff
@@ -75,15 +75,15 @@ that implements all requirements classes. The ResultSet class has no equivalent
     "http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/job-list",
     "http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/callback",
     "http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/dismiss",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/oaproc-routing",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/intermediate-waypoints",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/max-height",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/max-weight",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/obstacles",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/routing-engine",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/routing-algorithm",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/source-dataset",
-    "http://www.opengis.net/spec/ogcapi-routes-1/1.0/conf/time"
+    "http://www.opengis.net/spec/ogcapi-routes-1/1.0.0-draft.1/conf/oaproc-routing",
+    "http://www.opengis.net/spec/ogcapi-routes-1/1.0.0-draft.1/conf/intermediate-waypoints",
+    "http://www.opengis.net/spec/ogcapi-routes-1/1.0.0-draft.1/conf/max-height",
+    "http://www.opengis.net/spec/ogcapi-routes-1/1.0.0-draft.1/conf/max-weight",
```

Comment:
Shouldn't this be weight, instead of max-weight?

Reply:
I think for the name of the conformance classes, max height and weight makes sense. The property in the execution request is to specify the weight / height of the vehicle, but the capability supported by the routing engine is to consider the maximum weight / height of the road segments.

Reply:
@skyNacho - I agree with @jerstlouis.

Reply:
Yes, I understand now. It makes sense